pilot/pkg/networking/plugin/authn/authentication.go

```diff
@@ -194,7 +216,7 @@ func ConvertPolicyToAuthNFilterConfig(policy *authn.Policy) *authn_filter.Filter
 
 // BuildJwtFilter returns a Jwt filter for all Jwt specs in the policy.
 func BuildJwtFilter(policy *authn.Policy) *http_conn.HttpFilter {
-	filterConfigProto := ConvertPolicyToJwtConfig(policy)
+	filterConfigProto := ConvertPolicyToJwtConfig(policy, enableLocalJwks)
```

Comment:
You can use true here. Also add comment like `v2 api will use inline public key (jwksURI doesn't work as jwks clusters are not set up for v2)` to be clear

Reply:
done